wordpress/wp-content/mu-plugins/cds-base/classes/Modules/Users/Users.php

```diff
@@ -184,6 +184,25 @@ public function sendReset($uId, $email)
         wp_mail($email, $subject, $message);
     }
 
+    public function sendAddToCollection($uId, $email)
+    {
+        $userInfo = get_userdata($uId);
+
+        $blogName = get_bloginfo('name');
+
+        // phpcs:disable
+        $subject = __("Invitation to collaborate on GC Articles", "cds-snc"). " — ". $blogName;
+        
+        $blogInfo->name;
```

Comment:
This throws an error (and it's not used for anything anyway. It feels like it was left in by accident.)

Reply:
removed